docs/components/xdczero/design.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 1
+---
+
+# design
+
+Achieve unparalleled blockchain interoperability with XDC ZERO, ensuring frictionless data transmission and rigorous validation across a multitude of chains.
```

Comment:
XDC-Zero is a cross-chain framework that allows interoperability between XDC-Subnet and the XDC network. It ensures frictionless data transmission and rigorous validation across the Subnet and the Parentchain.